### PR TITLE
Getting started guide improvements

### LINKED ISSUE #262: Getting started guide improvements
In step 6, deploy skipper, there is a notice that one should uncomment properties in `src/kubernetes/server/server-deployment.yaml`  This should instead be described in the next step.

`Step 7.  Deploy the Data Flow Server` and  `Step 8. Deploy the Spring Cloud Data Flow Server for Kubernetes...` are redundant titles

These steps be combined or Step 7 should be about additional modifications to the yml files, while step 8 is just about shipping the yaml files to k8s.

### PR DESCRIPTION
**Pull request by chrisjs**
fixes #262

### CODE REVIEW

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -119,31 +119,51 @@ You can use the command `kubectl get all -l app=metrics` to verify that the depl
...
+    spec:
+      containers:
+      - name: skipper
+        image: springcloud/spring-cloud-skipper-server:1.0.1.RELEASE   # <1>
+        imagePullPolicy: Always
+----
++
+<1> You may change the version as you like.
++
+NOTE: Skipper includes the concept of link:https://docs.spring.io/spring-cloud-skipper/docs/current/reference/htmlsingle/#platforms[platforms],
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: #platforms is not present.  Did you mean https://docs.spring.io/spring-cloud-skipper/docs/current/reference/htmlsingle/#using-platforms?

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -119,31 +119,51 @@ You can use the command `kubectl get all -l app=metrics` to verify that the depl
...
+so it is important to define the "accounts" based on the project preferences. In the above YAML file, the accounts map
+to `minikube` as the platform. This can be modified, and of course, you can have any number of platform definitions.
+More details are in Spring Cloud Skipper reference guide.
 +
 
 [NOTE]
 ====
-To use Skipper, you _must_ uncomment the following properties to `src/kubernetes/server/server-deployment.yaml`. under the `env:` section
+If you'd like to orchestrate stream processing pipelines with Apache Kafka as the messaging middleware, you must change
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: Consider updating `as the messaging middleware` to `as the messaging middleware using Skipper`

**Review by cppwfs — COMMENTED** [review]
If you agree with the changes I can do them on merge.

**Comment by cppwfs** [comment]
LGTM Made a couple of comments.  If you agree with the changes I can do them on merge.

**Comment by chrisjs** [comment]
Updated the doc for both comments, thanks @cppwfs good to merge

**Pull request closed without merging**